src/preact/logger.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Use logger for intentionally logging messages in production code.
+ */
+export const logger = {
+  info: console.info.bind(console),
```

Comment:
Because you're using `.bind` here, these cannot be DCE'd. But, `console.info` (and the rest) doesn't require a `this` for it to work, so this can just be:
```suggestion
  info: console.info,
```

Reply:
Done